layers/+os/osx/config.el

```diff
@@ -35,10 +35,10 @@
    Default: `meta'.
    For backwards compatibility the variable `osx-use-option-as-meta'
    takes precedence is set to t.")
-(defvar osx-function-as 'none
+(defvar osx-function-as nil
   "Sets the key binding of the `FUCTION' key on OSX.
```

Comment:
The missing "N" in `FUCTION` can probably also be fixed when the documentation is updated for this PR.
With that change, the first line in the doc-string becomes:
"Sets the key binding of the `FUNCTION' key on OSX.